controllers/nc_news.controller.js

```diff
@@ -1,4 +1,9 @@
 const { selectTopics } = require("../model/nc_news.models");
+const endpoints = require("../endpoints.json")
+
+function getAPI(request, response) {
+    return response.status(200).send(endpoints)
```

Comment:
it would be nice to send this back on an object of endpoints like this:    {endpoints}